katdal/vis_flags_weights.py

```diff
@@ -161,6 +168,8 @@ def weight_power_scale(vis, weights, auto_indices, index1, index2, out=None):
         Chunk of weight data, with the same shape as `vis`.
     auto_indices, index1, index2 : np.ndarray
         Arrays returned by :func:`corrprod_to_autocorr`
+    divide : bool, optional
+        True if weights are divided by autocorrelations; otherwise they are multiplied
```

Comment:
It's not totally clear whether this refers to the input weights or the output weights. Maybe "should be divided"?

Reply:
I have hopefully now improved the docstring.